### PR TITLE
Remove global Mn, BB

### PR DESCRIPTION
**Pull request by denar90**
Added global $, Mn, Bb for skeleton

### CODE REVIEW

File: brunch/brunch-config.js [hunk 1]
```diff
@@ -12,6 +12,13 @@ module.exports = {
       joinTo: {'app.js': /^app\/templates/}
     }
   },
+  npm: {
+    globals: {
+      $: 'jquery',
+      Marionette: 'backbone.marionette',
+      Backbone: 'backbone.marionette/node_modules/backbone'
```
Comment (scott-w) [inline brunch/brunch-config.js]: Why use globals instead of explicitly importing the variables? I appreciate `$` can be unavoidable.

Comment (denar90) [inline brunch/brunch-config.js]: I just wanted to simplify app files using global Mn. Most of app files will use $, Mn, Bb and it's became annoying adding each time in top of them smth like `import Marionette from 'backbone.marionette`. That's why I proposed it)
What do you think, is it useless? Should we live it as it is?

**Comment by denar90** [comment]
@samccone @scott-w do I have to close this PR or smth?

**Comment by scott-w** [comment]
If you're ok with it as-is, then feel free.

Personally, I prefer forcing imports everywhere - it makes it clear what you're using and where.

**Comment by denar90** [comment]
Should I remove Mn and stuff from global scope in browserify and webpack examples? To have consistent way everywhere?

**Comment by scott-w** [comment]
A consistent approach would be best. If anyone else has a preference for global/non-global, feel free to chip in.

**Comment by denar90** [comment]
I've updated PR. I added travis to check if build for `browserify`, `brunch`, `webpack` going well
But it is needed to add "secure key" to `travis.yaml` to make it works.

**Comment by scott-w** [comment]
:+1:

**Comment by denar90** [comment]
@samccone can you look at this PR?

**Comment by ahumphreys87** [comment]
:+1:

**Pull request merged**